dev-server/ui-playground/shared/components/PlaygroundApp.js

```diff
@@ -68,6 +84,25 @@ export default function PlaygroundApp() {
             </li>
           ))}
         </ul>
+        {extraRoutes.length && (
+          <>
+            <h2>Application Patterns</h2>
+            <ul>
+              {extraRoutes.map(c => (
+                <li key={c.route}>
+                  <a
+                    className="PlaygroundApp__nav-link"
+                    key={c.route}
+                    href={/** @type string */ (c.route)}
```

Comment:
This assumes that `route` is a string here and not a regex, which is currently true. You could also create a type similar to `PlaygroundAppRoute` which only allows string routes and set the type of `extraRoutes` to be that.

Reply:
Good point: `extraRoutes` should use strings _only_ here.